src/helper/style-path.js

```diff
@@ -339,6 +350,33 @@ const applyStrokeWidthToSelection = function (value, textEditTargetId) {
     return changed;
 };
 
+const _colorStateFromGradient = gradient => {
+    const colorState = {};
+    // Scratch only recognizes 2 color gradients
+    if (gradient.stops.length === 2) {
+        if (gradient.radial) {
+            colorState.gradientType = GradientTypes.RADIAL;
+        } else {
+            // Always use horizontal for linear gradients, since horizontal and vertical gradients
+            // are the same with rotation. We don't want to show MIXED just because anything is rotated.
+            colorState.gradientType = GradientTypes.HORIZONTAL;
+        }
+        colorState.primary = gradient.stops[0].color.alpha === 0 ?
+            null :
+            gradient.stops[0].color.toCSS();
+        colorState.secondary = gradient.stops[1].color.alpha === 0 ?
+            null :
+            gradient.stops[1].color.toCSS();
+    } else {
+        if (gradient.stops.length < 2) log.warn(`Gradient has ${gradient.stops.length} stop(s)`);
+
+        colorState.primary = MIXED;
+        colorState.secondary = MIXED;
+    }
+
+    return colorState;
```

Comment:
Can we document the params of the return value, and note that gradientType may be null?